lib/stripe/core_resources/customer.ex

```diff
@@ -155,6 +155,7 @@ defmodule Stripe.Customer do
           {:ok, t} | {:error, Stripe.Error.t()}
   def delete_discount(id, opts \\ []) do
     new_request(opts)
+    |> prefix_expansions("data")
```

Comment:
✂️  I _think_ this could be removed since this just returns a deleted boolean. ✂️

Reply:
Well spotted! This was supposed to be in the list function above, now update.